### ISSUE TITLE
Can you insert an image in a form?

### ISSUE THREAD

**Issue by maasha** [kevinchappell/formBuilder#227]
I would like to add an explanatory image to a paragraph. Can it be done?

**Comment by kevinchappell** [comment]
Currently no but this may be implemented in the future. Adding to the backlog.

**Comment by Mateuil56** [comment]
Hello, has this option been added?

**Comment by eformx** [comment]
Convert your image to a base64 data image.  Drop a paragraph widget on Form Builder.  From the content text box add an img tag and set the src to the data string.  Works like a charm.

**Comment by Tabssummulani** [comment]
> This can be done now using custom controls.

Yes it woks fine ,but i am creating custom control like  'paragraph' and renamed as 'Image'  ,so i need to disabled-attribute of only custom control (Image).
how can i do?
if i apply
     disabledSubtypes: { paragraph: ["p"] }

so it will apply paragraph control , i want to hide subtype from custom control(Image) not inbuilt control
